modules/update-using-custom-machine-config-pools-inheritance.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-using-custom-machine-config-pools-mcp-inheritance_{context}"]
+= Managing machine configuration inheritance for a worker pool canary
+
+You can configure a machine config pool (MCP) canary to inherit any `MachineConfig` assigned to an existing MCP. 
```

Comment:
```suggestion
This configuration is useful when you want to use an MCP canary to test as you update nodes one at a time for an existing MCP.
```

Reply:
There were two lines here

You can configure a machine config pool (MCP) canary to inherit any MachineConfig assigned to an existing MCP. This configuration is useful when you want to use an MCP canary to test as you update the nodes of an existing MCP, one at a time.

I think you meant to replace the second one, rather than line 9.

Reply:
Yes. Not sure what happened here. My suggestion targets the second sentence.